hermes/crates/cbork/cddl-parser/src/lib.rs

```diff
@@ -7,23 +7,80 @@
 use std::fmt::Debug;
 
 pub use pest::Parser;
-use pest_derive::Parser;
 
 extern crate derive_more;
 use derive_more::{Display, From};
+use pest::error::Error;
 
-// Parser with DEBUG rules.  These rules are only used in tests.
-#[derive(Parser)]
-#[grammar = "grammar/cddl.pest"]
-#[grammar = "grammar/cddl_test.pest"] // Ideally this would only be used in tests.
-pub struct CDDLParser;
+pub mod rfc_8610 {
+    pub use pest::Parser;
+
+    #[derive(pest_derive::Parser)]
+    // #[grammar = "grammar/rfc_8610.pest"]
+    // TODO: we will implement to support those specifications later
+    #[grammar = "grammar/cddl.pest"]
+    pub struct RFC8610Parser;
+}
+
+pub mod rfc_9615 {
+    pub use pest::Parser;
+
+    #[derive(pest_derive::Parser)]
+    // #[grammar = "grammar/rfc_8610.pest"]
+    // #[grammar = "grammar/rfc_9615.pest"]
```

Comment:
Add this `.pest` file, but just put a comment in it and no grammar.
That way it can be properly referenced here already.